library/build.gradle

```diff
@@ -28,6 +28,8 @@ android {
     lintOptions {
         warningsAsErrors true
         abortOnError true
+        // We don't want to impose RTL on consuming applications.
+        disable 'RtlEnabled'
```

Comment:
Since we use attributes like `end` or `start` Lint complains about RTL not being enabled in the manifest. But this can be done only on the application level. This is related to this PR in a sense that we don't use `left` or `right` anymore.